apps/cowswap-frontend/src/modules/tokensList/containers/SelectTokenWidget/index.tsx

```diff
@@ -226,6 +233,10 @@ export function SelectTokenWidget({ displayLpTokenLists, standalone }: SelectTok
             tokenListTags={tokenListTags}
             areTokensFromBridge={areTokensFromBridge}
             isRouteAvailable={isRouteAvailable}
+            tradeType={tradeType}
+            field={field}
+            isDarkMode={darkMode}
```

Comment:
It's better not to drill down `isDarkMode` prop. I see it's used only in `ChainLogo` component.
`useTheme()` can be used in pure components.

Reply:
Updated. ChainLogo.tsx now calls useTheme() directly